src/components/TodoItem/TodoItem.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+const TodoItem = (props) => {
+  function handleCheck() {
```

Comment:
rewrite as arrow functions, you already used this syntax above so do the same for each next func